nebula/ui/components/VPNCheckBoxRow.qml

```diff
@@ -73,6 +74,11 @@ RowLayout {
             text: labelText
             color: VPNTheme.theme.fontColorDark
             horizontalAlignment: Text.AlignLeft
+
+            VPNMouseArea {
+                propagateClickToParent: false
```

Comment:
I like this but, not always we have a parent-child relationship between QML items.
What about if we implement a generic `VPNMouseAreaProxy` QML component with a: `propagateTo` property (or a better name). We can use the `propagateTo` property to link QML items.
Thoughts?

Reply:
Interesting. My question is, can't we just have whatever functionality we need in the `onClicked()` handler? Wouldn't that leave it more flexible (i.e. you can propagate the click to multiple other components if you wanted to)?